ci/fats.sh

```diff
@@ -60,13 +54,7 @@ travis_fold end system-install
 # run test functions
 source $fats_dir/functions/helpers.sh
 
-if [ "$mode" = "full" ]; then
-  functions=(java java-boot node npm command)
-else
-  functions=(command)
-fi
-
-for test in "${functions[@]}"; do
+for test in command; do
```

Comment:
Since we are not running in lite mode, should we run `java java-boot node npm` too?

Reply:
The cli knows nothing about the content of the function. All running with multiple functions does is test the same flow multiple times.

Reply:
ok, I suppose it is more valuable for cnab-riff to run all functions before putting out a build for our users.

Reply:
the most important place to test all of the functions is before releasing the builder (we test all of the functions here already). The second most important place is when releasing the bundle.